src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_enkf_update.F90

```diff
@@ -2738,10 +2738,10 @@ subroutine write_smapL4SMaup( option, date_time, work_path, exp_id, N_ens,    &
                 data_h_9km_tile_8 = real(nodata_generic,kind(0.0D0)) ! init (not in grid2tile!)
                 data_v_9km_tile_8 = real(nodata_generic,kind(0.0D0)) ! init (not in grid2tile!)
 
-                call grid2tile( tile_grid_g, N_catf, tile_coord_f%i_indg, tile_coord_f%j_indg, data_h_9km_grid_8, &
+                call grid2tile( tile_grid_g, N_catf, tile_coord_f%hash_i_indg, tile_coord_f%hash_j_indg, data_h_9km_grid_8, &
```

Comment:
I'm pretty sure this should be %i_indg, not %hash_i_indg.  Here, we are simply going back and forth between the tile space and the grid that is associated with this tile space.  Because this subroutine only works for the EASE grid, and because %i_indg and %hash_i_indg are identical in this case, the results are going to be the same.  But we still need to use the appropriate field of the tile_coord_f structure.